Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1534,6 +1534,66 @@
 /* Title for the 'scan QR code' view. */
 "SCAN_QR_CODE_VIEW_TITLE" = "Scan QR Code";
 
+/* Indicates a delay of zero seconds, and that 'screen lock activity' will timeout immediately. */
+"SCREEN_LOCK_ACTIVITY_TIMEOUT_NONE" = "None";
+
+/* Title for alert indicating that screen lock could not be disabled. */
+"SCREEN_LOCK_DISABLE_FAILED" = "Screen Lock Could Not Be Disabled";
+
+/* Title for alert indicating that screen lock could not be enabled. */
+"SCREEN_LOCK_ENABLE_FAILED" = "Screen Lock Could Not Be Enabled";
+
+/* Indicates that an unknown error occurred while using Touch ID or Face ID. */
+"SCREEN_LOCK_ENABLE_UNKNOWN_ERROR" = "Touch ID and Face ID could not be accessed.";
+
+/* Indicates that Touch ID/Face ID authentication failed. */
+"SCREEN_LOCK_ERROR_LOCAL_AUTHENTICATION_FAILED" = "Authentication with Touch ID or Face ID failed.";
+
+/* Indicates that Touch ID/Face ID is 'locked out' on this device due to authentication failures. */
+"SCREEN_LOCK_ERROR_LOCAL_AUTHENTICATION_LOCKOUT" = "Touch ID or Face ID is locked out; too many failures.";
+
+/* Indicates that Touch ID/Face ID are not available on this device. */
+"SCREEN_LOCK_ERROR_LOCAL_AUTHENTICATION_NOT_AVAILABLE" = "Touch ID or Face ID is not available on this device.";
+
+/* Indicates that Touch ID/Face ID is not configured on this device. */
+"SCREEN_LOCK_ERROR_LOCAL_AUTHENTICATION_NOT_ENROLLED" = "Touch ID or Face ID is not configured.";
+
+/* Indicates that Touch ID/Face ID passcode is not set. */
+"SCREEN_LOCK_ERROR_LOCAL_AUTHENTICATION_PASSCODE_NOT_SET" = "Touch ID or Face ID is not available; no passcode has been set.";
+
+/* Description of how and why Signal iOS uses Touch ID/Face ID to disable 'screen lock'. */
+"SCREEN_LOCK_REASON_DISABLE_SCREEN_LOCK" = "Use Touch ID or Face ID to disable Screen Lock.";
+
+/* Description of how and why Signal iOS uses Face ID to disable 'screen lock'. */
+"SCREEN_LOCK_REASON_DISABLE_SCREEN_LOCK_FACE_ID" = "Use Face ID to disable Screen Lock.";
+
+/* Description of how and why Signal iOS uses Touch ID to disable 'screen lock'. */
+"SCREEN_LOCK_REASON_DISABLE_SCREEN_LOCK_TOUCH_ID" = "Use Touch ID to disable Screen Lock.";
+
+/* Description of how and why Signal iOS uses Touch ID/Face ID to enable 'screen lock'. */
+"SCREEN_LOCK_REASON_ENABLE_SCREEN_LOCK" = "Use Touch ID or Face ID to lock access to Signal.";
+
+/* Description of how and why Signal iOS uses Face ID to enable 'screen lock'. */
+"SCREEN_LOCK_REASON_ENABLE_SCREEN_LOCK_FACE_ID" = "Use Face ID to lock access to Signal.";
+
+/* Description of how and why Signal iOS uses Touch ID to enable 'screen lock'. */
+"SCREEN_LOCK_REASON_ENABLE_SCREEN_LOCK_TOUCH_ID" = "Use Touch ID to lock access to Signal.";
+
+/* Description of how and why Signal iOS uses Touch ID/Face ID to unlock 'screen lock'. */
+"SCREEN_LOCK_REASON_UNLOCK_SCREEN_LOCK" = "Use Touch ID or Face ID to unlock Screen Lock.";
```

Comment:
Use Touch ID or Face ID to access Signal.

Reply:
I'm going with your other suggestion: "Authenticate to access Signal".